modules/dnn/test/test_layers.cpp

```diff
@@ -777,6 +777,50 @@ TEST(Layer_PriorBox, squares)
     normAssert(out.reshape(1, 4), target);
 }
 
+TEST(Layer_Test_DWconv_Prelu, Accuracy)
+{
+    // Test case
+    // input       img size 3x16x16  value all 1
+    //   |
+    //   v
+    // dw_conv     weight[0]=-1 weight[1]=-2 weight[2]=-3 no_bias
+    //   |
+    //   v
+    // prelu       weight={1,2,3}
+    //   |
+    //   v
+    // output      out size 3x14x14  if right: out[0]=-9 out[0]=-36 out[0]=-81 
+    //             but current opencv output: out[0]=-27 out[0]=-54 out[0]=-81
+
+    Net net;
+    {
+        const string proto = findDataFile("dnn/conv_prelu.prototxt", false);
+        const string model = findDataFile("dnn/conv_prelu.caffemodel", false);
+        net = readNetFromCaffe(proto, model);
+        ASSERT_FALSE(net.empty());
+    }
+    Mat sample = imread(findDataFile("dnn/3_16_16_all_1.jpg", false));
+    ASSERT_TRUE(!sample.empty());
+
+    if (sample.channels() == 4)
+        cvtColor(sample, sample, COLOR_BGRA2BGR);
+
+    Mat in_blob = blobFromImage(sample, 1.0f, Size(16, 16), Scalar(), false, false);
```

Comment:
You can generate an input blob by creating a `cv::Mat` with a shape `1x3x16x16` and fill it by ones.
In example, 
```cpp
int shape[] = {1, 3, 16, 16};
Mat in_blob(4, &shape[0], CV_32FC1, Scalar(1));
```

Reply:
ok, let me modify it.